### ISSUE TITLE
Scanner FieldAnnotationsScanner was not configured

### ISSUE THREAD

**Issue by diegomarzo** [JetBrains/kotless#38]
Looks like in the new version of Kotless a bug has been introduced in relation with the reflection capabilities.

With a simple code like:

```
import io.kotless.dsl.lang.http.Get

@Get("/hello")
fun sayHello() = "Say Hello!"
```

When the endpoint is hit the following exception is thrown:

```
org.reflections.ReflectionsException: Scanner FieldAnnotationsScanner was not configured
        at org.reflections.Store.get(Store.java:39)
        at org.reflections.Store.get(Store.java:61)
        at org.reflections.Store.get(Store.java:46)
        at org.reflections.Reflections.getFieldsAnnotatedWith(Reflections.java:546)
        at io.kotless.dsl.reflection.ReflectionScanner.fieldsWithAnnotation(ReflectionScanner.kt:43)
        at io.kotless.local.handler.StaticHandler$fields$2.invoke(StaticHandler.kt:38)
        at io.kotless.local.handler.StaticHandler$fields$2.invoke(StaticHandler.kt:14)
        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
        at io.kotless.local.handler.StaticHandler.getFields(StaticHandler.kt)
        at io.kotless.local.handler.StaticHandler.handle(StaticHandler.kt:19)
        at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:59)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.Server.handle(Server.java:501)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
        at java.base/java.lang.Thread.run(Thread.java:832)
```

Going back to version 0.1.3 solves the problem (using diverse versions of Kotlin: 1.3.61 and 1.3.72)

Note: The problem is reproducible in local

**Comment by TanVD** [comment]
Wow, strange that it was not reproduced during my tests.

**Comment by TanVD** [comment]
Hm, for some reason with my test examples I am unable to reproduce it. Could you please send me a zip with a project, so I could test it? And check that you have updated not only plugin, but also DSL version. 
In code FieldsAnnotationScanner initialized, but I've updated `reflections` version to 0.9.12, so maybe it is a regression in library.

**Comment by diegomarzo** [comment]
Sure!

I have invited you to the a repository named `kotless-server`. It only has the function I mention. At the moment works with version 0.1.3

Thank you for your amazing work by the way.

**Comment by TanVD** [comment]
Yay, it reproduced. Working on fix. Probably will release it today/tomorrow in 0.1.5

**Comment by TanVD** [comment]
Yep, actual culprit is update to reflections:0.9.12 -- https://github.com/ronmamo/reflections/issues/273 .  For now I'll revert this update, but will consider move to another library

**Comment by TanVD** [comment]
Fix released with version 0.1.5
